src/bin/cql-stress-cassandra-stress/settings/command/mod.rs

```diff
@@ -77,31 +77,24 @@ impl Command {
     }
 }
 
-pub enum CommandParams {
-    // HELP, PRINT, VERSION
-    Special,
-
-    // READ, WRITE, COUNTER_READ, COUNTER_WRITE
-    BasicParams(ReadWriteParams),
-    // MIXED
-    // TODO: MixedParams,
-
-    // USER
-    // TODO: UserParams,
+pub struct CommandParams {
+    // Parameters shared across all of the commands
+    pub basic_params: ReadWriteParams,
+    // TODO:
+    // mixed_params: Option<MixedParams>
+    // user_params: Option<UserParams>
```

Comment:
If you look at the parameters of some commands, you can see that parameters of `read/write` are a subset of parameters of both `mixed` and `user` command. IMO, it makes sense to get rid of the enum, and represent it as struct, since both `mixed` and `user` will include `basic_params`, and some additional parameters specific to them.

Reply:
Ok. Again, this should be mentioned in the commit message.